components/brave_new_tab_ui/components/default/rewards/style.ts

```diff
@@ -0,0 +1,150 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+import palette from 'brave-ui/theme/colors'
+
+export const WidgetWrapper = styled<{}, 'div'>('div')`
+  color: white;
+  padding: 10px 15px;
+  border-radius: 6px;
+  position: relative;
+  font-family: Muli, sans-serif;
+  overflow-x: hidden;
+  background-image: linear-gradient(140deg, #392DD1 0%, #8E2995 100%);
+`
+
+export const Footer = styled<{}, 'div'>('div')`
+  max-width: 275px;
+  margin-top: 10px;
+`
+
+export const BatIcon = styled<{}, 'div'>('div')`
+  width: 30px;
+  height: 30px;
+  display: inline-block;
+`
+
+export const RewardsTitle = styled<{}, 'span'>('span')`
+  top: -9px;
+  left: 8px;
+  font-size: 16px;
+  font-weight: 600;
+  position: relative;
+`
+
+export const ServiceText = styled<{}, 'span'>('span')`
+  color: rgba(255, 255, 255, 0.65);
+  font-size: 14px;
+  font-weight: normal;
+  letter-spacing: 0;
+  line-height: 18px;
+`
+
+export const ServiceLink = styled<{}, 'a'>('a')`
+  color: ${palette.blue500};
+  font-weight: 600;
+  text-decoration: none;
+`
+
+export const LearnMoreLink = styled(ServiceLink)`
+  margin-right: 5px;
+`
+
+export const LearnMoreText = styled<{}, 'div'>('div')`
+  margin-top: 50px;
+`
+
+export const PreOptInInfo = styled<{}, 'div'>('div')`
+  margin-top: 20px;
+`
+
+export const Title = styled<{}, 'span'>('span')`
+  font-size: 20px;
+  font-weight: 600;
+  display: block;
+`
+
+export const SubTitle = styled<{}, 'span'>('span')`
+  font-size: 16px;
+  font-weight: 600;
+  display: block;
+  margin-top: 10px;
+  max-width: 250px;
+`
+
+export const PreOptInAction = styled<{}, 'div'>('div')`
+  margin-top: 30px;
+  text-align: center
+`
+
+export const TurnOnButton = styled<{}, 'button'>('button')`
+  border-radius: 20px;
+  background: white;
+  color: ${palette.blurple500};
+  font-weight: bold;
+  font-size: 16px;
+  padding: 7px 60px;
+  margin: 0 auto;
+  cursor: pointer;
+`
+
+export const TurnOnAdsButton = styled(TurnOnButton)`
+  width: 100%;
+  margin-bottom: 5px;
+  display: block;
+`
+
+export const AmountItem = styled<{}, 'div'>('div')`
+  margin-top: 10px;
+`
+
+export const AmountInformation = styled<{}, 'div'>('div')`
+  margin-bottom: 5px;
+`
+
+export const Amount = styled<{}, 'span'>('span')`
+  font-size: 22px;
+  margin-right: 10px;
+`
+
+export const ConvertedAmount = styled<{}, 'span'>('span')`
+  font-size: 16px;
+`
+
+export const AmountDescription = styled<{}, 'span'>('span')`
+  font-size: 16px;
+  color: ${palette.blue500};
+  font-weight: 600;
+`
+
+export const NotificationWrapper = styled(WidgetWrapper)`
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  box-shadow: 5px 10px 8px 10px #000;
+`
+
+export const NotificationButton = styled(TurnOnButton)`
+  background: #24B028;
```

Comment:
what's this color? Designer's confirmed this can't be palette color?

Reply:
now using theme palette color